contributors/devel/release/issues.md

```diff
@@ -0,0 +1,185 @@
+# Targeting issues and PRs to release milestones
+
+This document describes how to target issues and PRs to a release.
+The process for shepherding issues into the release by the owner, release team, and GitHub bot
+is outlined below.
+
+## Definitions
+
+- *issue owners*: creator, assignees, and user who moved the issue into a release milestone
+- *Y days*: refers to business days (using the location local to the release-manager M-F)
+- *code slush*: starts when master branch only accepts PRs for release milestone.  no additional feature development is merged after this point.
+- *code freeze*: starts 2 weeks after code slush.  only critical bug fixes are accepted into the release codebase.
+
+## Requirements for adding an issue to the milestone
+
+**Note**: Issues with unmet label requirements will automatically be removed from the release milestone.
+
+When adding an issue to a milestone, the Kubernetes bot will check that the following
+labels are set, and comment on the issue with the appropriate instructions.  The
+bot will attempt to contact the issue creator 3 times (over 3 days)
+before automatically removing the issue from the milestone.
+
+Label categories:
+
+- SIG label owner
+- Priority
+- Issue type
+
+### SIG owner label
+
+The SIG owner label defines the SIG to which the bot will escalate if the issue is not resolved
+or updated by the deadline.  If there are no updates after escalation, the
+issue may automatically removed from the milestone.
+
+e.g. `sig/node`, `sig/federation`, `sig/apps`, `sig/network`
+
+### Priority
+
+Priority label used by the bot to determine escalation path before moving an issues
+out of the release milestone.  Also used to determine whether or not a release should be
+blocked on the resolution of the issue.
+
+- `priority/critical-urgent`: Never automatically move out of a release milestone; continually escalate to contributor and SIG through all available channels.
+  - considered a release blocking issue
+  - post code slush: issue owner update frequency: every 3 days
+  - code freeze: issue owner update frequency: daily
+  - would require a patch release if left undiscovered until after the minor release.
+- `priority/important-soon`: Escalate to the issue owners and SIG owner; move out of milestone after several unsuccessful escalation attempts.
+  - not considered a release blocking issue
+  - would not require a patch release
```

Comment:
Should this line and the one below also apply to ``priority/important-longterm``?  So the only difference between the 2 would be the mount of escalation?